pkg/provider/gke/gke.go

```diff
@@ -426,6 +426,33 @@ func (c *GKE) nodePoolRunning(zone, projectID, clusterID, poolName string) (bool
 	return false, nil
 }
 
+// NodePoolCheck returns an error if any of the nodepool do not exist.
+func (c *GKE) NodePoolCheck(*kingpin.ParseContext) error {
+	reqC := &containerpb.CreateClusterRequest{}
+
+	exists := true
```

Comment:
should this be more clear as 
```suggestion
	allNodepoolsRunning := true
```